src/site/_data/authorsData.json

```diff
@@ -1467,6 +1467,20 @@
     },
     "image": "image/admin/fbCJc4uifsZ4d5KolJ7j.jpg"
   },
+  "foolip": {
+    "name": {
+      "given": "Philip",
+      "family": "Jägenstedt"
+    },
+    "github": "foolip",
+    "twitter": "foolip",
+    "homepage": "https://foolip.org/",
+    "org": {
+      "name": "Google",
+      "unit": "Chrome"
+    },
+    "image": "TODO"
```

Comment:
How should I provide the image? https://foolip.org/cv/headshot-2018.jpg is the source image.

Reply:
Use the uploader at this link https://web-dev-uploads.web.app/uploader and copy/paste the image shortcode you get into the `image` field.

Reply:
Ah, thanks!